source/incident-collaboration/running-playbooks.rst

```diff
@@ -18,24 +18,24 @@ The ``/playbook`` slash command allows interaction with incidents via the post t
 - ``/playbook owner [@username]`` - Show or change the current playbook run owner.
 - ``/playbook info`` - Show a summary of the current playbook's activities.
 
-To run a playbook use the ``/playbook start`` slash command from any channel. You can also use the desktop or browser to:
+To run a playbook, use the ``/playbook start`` slash command from any channel. You can also use the desktop or browser to:
 
  * Select the **Playbook** icon in the channel header, and select **Run playbook** from the right-hand sidebar.
  * Use the context menu of a post and select **Run playbook**.
 
-Running a playbook opens an interactive dialog. You can create a new playbook or select an existing playbook. Provide a name for your playbook run, then select **Start run**. Only playbooks you belong to are listed in the **Playbook** drop-down menu. The creator of a playbook run is automatically added as the first member and becomes the owner. Lastly, when the playbook run is created, it's announced in the associated channel by the Playbook Bot.
+Running a playbook opens an interactive dialogue. You can create a new playbook or select an existing playbook. Provide a name for your playbook run, then select **Start run**. Only playbooks you belong to are listed in the **Playbook** drop-down menu. The creator of a playbook run is automatically added as the first member and becomes the owner. Lastly, when the playbook run is created, it's announced in the associated channel by the Playbook Bot.
```

Comment:
In this context, ``dialog`` is the correct word to use. The term ``dialog`` is used in computing contexts, and ``dialogue`` is used in contexts related conversation and the exchange of ideas.

Reply:
okay understood